src/QuizAtom.tsx

```diff
@@ -1,7 +1,268 @@
-import { QuizAtomType } from './types';
+import {
+    QuizAtomType,
+    Question as QuestionType,
+    Answer as AnswerType,
+} from './types';
 
-export const QuizAtom = ({ id }: QuizAtomType): JSX.Element => (
-    <div data-atom-id={id}>QuizAtom</div>
-);
+const neutralBackgroundColour = css`
+    background-color: ${neutral[97]};
+`;
+const darkerNeutralBackgroundColour = css`
+    background-color: ${neutral[86]};
+`;
+const opaqueGreenBackgroundColour = css`
+    background-color: rgb(61, 181, 64);
+`;
+const translucentGreenBackgroundColour = css`
+    background-color: rgb(61, 181, 64, 0.6);
+`;
+const redBackgroundColour = css`
+    background-color: ${news[400]};
+`;
+
+const commonLabelStyle = css`
+    svg {
+        width: 30px;
+        height: 30px;
+    }
+    display: block;
+    padding: 0.75rem 1.25rem;
+    transition: background-colour 100ms;
+`;
+
+const focusLabelStyle = css`
+    cursor: pointer;
+    ${neutralBackgroundColour};
+    :hover {
+        ${darkerNeutralBackgroundColour};
+    }
+    :focus {
+        ${focusHalo}
+        ${darkerNeutralBackgroundColour};
+    }
+`;
+
+const commonTextStyle = css`
+    padding-left: 8px;
+    padding-top: 0;
+    min-height: auto;
+`;
+
+const blackColour = css`
+    color: ${neutral[7]};
+`;
+const whiteColour = css`
+    color: ${neutral[100]};
+`;
+const revealTextStyle = css`
+    ${textSans.xsmall()}
+`;
+
+const fieldsetStyle = css`
+    margin-bottom: 12px;
+    border: 0px;
+    padding: 0px;
+`;
+
+const questionStyle = css`
+    display: flex;
+    div {
+        padding-right: 0.5ch;
+    }
+    legend {
+        margin-bottom: 12px;
+    }
+    span {
+        padding-right: 12px;
+    }
+`;
+
+const answerStyle = css`
+    display: flex;
+    div {
+        display: flex;
+        flex-direction: column;
+    }
+`;
+
+const iconStyle = css`
+    height: 26px;
+    svg {
+        fill: ${neutral[100]};
+        height: 26px;
+        width: 26px;
+        view-box: '0 0 26px 26px';
+    }
+`;
+
+const IconWrapper = (el: JSX.Element) => <div className={iconStyle}>{el}</div>;
+
+export const QuizAtom = ({ id, questions }: QuizAtomType): JSX.Element => {
+    console.log(`rendered atom`);
+    return (
+        <div data-atom-id={id}>
+            <form>
+                {questions.map((question, idx) => (
+                    <Question
+                        key={question.id}
+                        id={question.id}
+                        number={idx + 1}
+                        text={question.text}
+                        imageUrl={question.imageUrl}
+                        answers={question.answers}
+                    />
+                ))}
+            </form>
+        </div>
+    );
+};
+
+export type QuestionProps = {
+    number: number;
+};
+
+export const Question = ({
+    text,
+    imageUrl,
+    answers,
+    number,
+}: QuestionType & QuestionProps): JSX.Element => {
+    const [chosen, setChosen] = useState<string | undefined>(undefined);
```

Comment:
I think the variable `chosen` might be better named to `selectedId`, I suspect this is an ID because you use `chosen === answer.id` later down the component?